src/main/java/DirectoryLister.java

```diff
@@ -0,0 +1,56 @@
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class DirectoryLister {
+    private int sum = 0;
+    private Scanner sc;
+
+    public void closeScanner() {
+        sc.close();
+    }
+
+    public String giveAddress() {
```

Comment:
method name should be getFolderPath since we are getting it from the user and we are getting path not address.